### ISSUE TITLE
Content Review Request: Toxic Exposure, from Disability Benefits Experience Team 1

### ISSUE THREAD

**Issue by mengA6** [department-of-veterans-affairs/va.gov-team#76216]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: `Disability Benefits Experience Team 1`
- **OCTO product owner:** @emilytheis 
- **OCTO design lead:**  @shannonkford 
- **Product name:** `Disability Benefits Form 21-526EZ`
- **Product manager:**  @RakshindaAslam 
- **Delivery manager:** @alatempa24 
- **Front end engineer:** @christinec-fftc 
- **Researcher:** @evelynhilbrichdavis 
- **Slack channel:** [benefits-disability-design](https://dsva.slack.com/archives/C053UDWMH7U)
- **Dedicated content writer on your team (if you have one):** `none`
- **Dedicated a11y specialist on your team (if you have one):** `none`
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

We are seeking a focused content review on some questions we will be updating in the disability benefits form as part of adding toxic exposure questions. There is [this screen](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=411-6431&mode=design&t=8OJjeOf2ZrzZEBwS-4) with the question "How is your condition related to your service?" and 4 radio button choices. Following from each of the 4 choices, there is another screen with questions customized to that radio button choice. These 4 screens are the ones we would like content review on. 

1. [Condition follow up option 1](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=0-1132&mode=design&t=8OJjeOf2ZrzZEBwS-4)
2. [Condition follow up option 2](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=699-7159&mode=design&t=8OJjeOf2ZrzZEBwS-11)
3. [Condition follow up option 3](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=699-10665&mode=design&t=8OJjeOf2ZrzZEBwS-11)
4. [Condition follow up option 4](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=699-11332&mode=design&t=8OJjeOf2ZrzZEBwS-11)

We know that there has been guidance to remove the word "please" in questions, so we have done that, and we also adapted the question text into current design system components, using "hint text" where it seems appropriate. We want to make sure we have interpreted this correctly and see if there are any other content edits you recommend before implementing these screens. @katherine-fung has given us content guidance in the past and is familiar with this work.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

Finalizing design adjustments out of a [second midpoint review](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/60140), beginning implementation. 

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent 
- [x] Midpoint review (twice)
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `60140`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/toxic-exposure.md) - some content in "background on decisions" section needs to be updated
- [Previous CAIA Intake ticket](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/71168)
- [Figma screens](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=0-1132&mode=design&t=8OJjeOf2ZrzZEBwS-11)
- [Figma prototype](https://www.figma.com/proto/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?page-id=0%3A411&type=design&node-id=0-1296&viewport=-965%2C-1306%2C0.28&t=cbkgDhMeRzrXJzk7-1&scaling=min-zoom&starting-point-node-id=0%3A1296&show-proto-sidebar=1&mode=design)

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [x] **If you are requesting content support**, please share your content source of truth : [Figma screens](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=0-1132&mode=design&t=8OJjeOf2ZrzZEBwS-11)
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```
